packages/components/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # @baseapp-frontend/components
 
+## 0.0.23
+
+### Patch Changes
+
+- Use jotai global state and cookies to keep track of the current profile. CurrentProfileProvider is removed, use "const { currentProfile, setCurrentProfile } = useCurrentProfile()" to read and modify the current profile, respectively.
```

Comment:
I think we need to update that

Reply:
What do you mean? Update the text? Or make this a minor/major version bump?